python/dlr/dlr_model.py

```diff
@@ -160,15 +164,42 @@ def get_input_names(self):
         """
         return self.input_names
 
+    def has_metadata(self) -> bool:
+        flag = ctypes.c_bool()
+        _check_call(_LIB.GetDLRHasMetadata(byref(self.handle), byref(flag)))
+        return flag.value
+    
+    def _fetch_output_names(self):
+        self.output_names = []
+        if self.has_metadata():
+            for i in range(self.num_outputs):
+                name = c_char_p()
+                _check_call(_LIB.GetDLROutputName(byref(self.handle), i, byref(name)))
+                self.output_names.append(name.value.decode('utf-8'))
+
+    def _fetch_input_names(self):
+        for i in range(self.num_inputs):
+            self.input_names.append(self._get_input_name(i))
+        
+    def _fetch_output_dtypes(self):
+        self.output_dtypes = []
+        if self.has_metadata():
```

Comment:
Metadata is needed only fo fetch Original Output names

Reply:
Yes. Is that a question of does this require a change?

Reply:
`if self.has_metadata()` should be removed

Reply:
Yes It can be removed 👍